x-pack/plugins/canvas/public/components/workpad_page/event_handlers.js

```diff
@@ -126,30 +126,6 @@ const handleMouseDown = (commit, e, isEditable) => {
 
 const keyCode = key => (key === 'Meta' ? 'MetaLeft' : 'Key' + key.toUpperCase());
 
-const isTextInput = ({ tagName, type }) => {
-  // input types that aren't variations of text input
-  const nonTextInputs = [
-    'button',
-    'checkbox',
-    'color',
-    'file',
-    'image',
-    'radio',
-    'range',
-    'reset',
-    'submit',
-  ];
-
-  switch (tagName.toLowerCase()) {
-    case 'input':
-      return !nonTextInputs.includes(type);
-    case 'textarea':
-      return true;
-    default:
-      return false;
-  }
-};
-
 const modifierKey = key => ['KeyALT', 'KeyCONTROL'].indexOf(keyCode(key)) > -1;
 
 const handleKeyDown = (commit, e, isEditable) => {
```

Comment:
I haven't cleaned up some code here because the expectation was that, we're going to add back the initially present layout keyboard shortcuts for moving/rotating/resizing elements, which were initially fully handled in aero. Since we decided that keystrokes should be none of aero's concern, I recently removed the processing code, and I think this `handleKeyDown` is no longer doing anything useful so could be removed. May be extra safe to search for `keyboardEvent` under `aeroelastic/`. A bit tangential to your PR but you're reworking a good bunch of key handling so removing this vestige may free you up for whatever this may have blocked (or just remove a few lines).

Reply:
I checked, and there aren't any instances of `keyboardEvent` in the aeroelastic code. I'll go ahead and remove the key event handlers